packages/lesswrong/server/dialogues/dialogueHelperBot.ts

```diff
@@ -0,0 +1,193 @@
+import {renderToString} from "react-dom/server";
```

Comment:
Is there anything new/changed in this file that wasn't just moved from `dialogueMatchPreferenceCallbacks.ts`?